pkg/services/authz/client.go

```diff
@@ -129,7 +130,7 @@ func newGrpcLegacyClient(address string) (authzlib.MultiTenantClient, error) {
 			grpc.WithStreamInterceptor(clientInterceptor.StreamClientInterceptor),
 		),
 		// nolint:staticcheck
-		authzlib.WithNamespaceFormatterLCOption(authnlib.OnPremNamespaceFormatter),
+		authzlib.WithNamespaceFormatterLCOption(claims.OrgNamespaceFormatter),
```

Comment:
These changes are due to authlib's changes, unrelated to this PR, but needed for building